modin/pandas/indexing.py

```diff
@@ -551,29 +543,35 @@ def __getitem__(self, key):
         if isinstance(result, Series):
             result._parent = self.df
             result._parent_axis = 0
+        col_loc_as_list = [col_loc] if is_scalar(col_loc) else col_loc
+        row_loc_as_list = [row_loc] if is_scalar(row_loc) else row_loc
```

Comment:
shouldn't we use here `self.col_scaler` and `self.row_scaler` computed above to know whether certain `loc` is scalar, instead of computing this info again via `is_scalar`?